guides/common/modules/proc_exporting-syncable-imports-via-a-web-server.adoc

```diff
@@ -0,0 +1,10 @@
+[id="Exporting_Syncable_Imports_via-a_Web-Server_{context}"]
+= Exporting Syncable Imports via a Web Server
+
+.Procedure
+* You can pass a HTTP URL to the path command :
++
+[options="nowrap" subs="+quotes"]
+----
+# CDN connected Sat |   | => [webserver] <copy URL here> -> 
```

Comment:
I find this syntax kind of unclear. Is there a ticket maybe where I could look at to get more info?

Reply:
@adamlazik1 I have asked @parthaa to take a look at this one since he's the SME on it.